scality_manila_utils/cli.py

```diff
@@ -0,0 +1,190 @@
+# Copyright (c) 2015 Scality
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import grp
+import logging
+import logging.handlers
+import os
+import pwd
+
+from scality_manila_utils.helper import Helper
+from scality_manila_utils.exceptions import EnvironmentException
+
+log = logging.getLogger(__name__)
+
+
+def setup_logger():
+    """
+    Setup root log handler.
+    """
+    log_format = logging.Formatter(
+        '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+    )
+    handler = logging.handlers.SysLogHandler('/dev/log')
```

Comment:
Any exceptions raised, and an accompanying stack trace will be available on the manila side, as it is not supressed, but bubbled up to the command line. I'm not sure that it adds much extra value esssentially log everything over ssh.